pkg/model/file/funcmap.go

```diff
@@ -26,5 +28,15 @@ func DefaultFuncMap() template.FuncMap {
 	return template.FuncMap{
 		"title": strings.Title,
 		"lower": strings.ToLower,
+		"hash":  hash,
```

Comment:
Nit: specify that this is a non-cryptographic hash function, for clarity's sake
```suggestion
		"hashFNV":  hashFNV,
```